ios/ci_scripts/ci_pre_xcodebuild.sh

```
@@ -0,0 +1,45 @@
+brew install getsentry/tools/sentry-cli
+
+# Export auth token
+export SENTRY_AUTH_TOKEN=$SENTRY_AUTH_TOKEN
+
+# Check multiple possible dSYM locations
+POSSIBLE_DSYM_PATHS=(
+    "/Volumes/workspace/DerivedData/Build/Products/Release-iphoneos"
+    "/Volumes/workspace/build.xcarchive/dSYMs"
+    "${CI_DERIVED_DATA_PATH}/Build/Products/Release-iphoneos"
+    "/Volumes/workspace/repository/ios/build"
+)
```

Shellcheck

[warning] In POSIX sh, arrays are undefined.

(SC3030)